chromium/product.yml

```diff
@@ -2,6 +2,6 @@ product: chromium
 full_name: Chromium
 type: product
 
-benchmark_root: "./guide"
+benchmark_root: ["./guide"]
```

Comment:
Let's remove the brackets and make this a comma separated list having the python code convert it into a list.

Reply:
Any particular reason for not using existing YAML format for lists and going imaginative here?

Reply:
evgenyz++, if YAML format is used, then no Python code is needed, because the conversion is done by pyyaml library.

Reply:
Then this needs to be the following:
```
benchmark_root: 
    - "./guide"
    - "./other_root"

Reply:
No, not necessarily. The `benchmark_root: ["./guide","./other_root"]` abbreviated form is legitimate YAML as well.